### PR TITLE
feat: Add GetSeverity(), GetCategory() for annots

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Relates to https://github.com/kubewarden/kubewarden-controller/issues/461

## Test

No need.

## Additional Information

PR against feat-audit, which we are consuming in audit-scanner.

### CODE REVIEW

**Comment by viccuad** [comment]
I tried to document the new annotations in the crd generation docs, but alas, I found no way to get the metadata annotations listed.

**Review by jvanz — APPROVED** [review]

**Comment by jvanz** [comment]
> I tried to document the new annotations in the crd generation docs, but alas, I found no way to get the metadata annotations listed.

Let's add something in our docs website then.

File: pkg/apis/policies/v1/policy.go [hunk 1]
```diff
@@ -98,4 +103,6 @@ type Policy interface {
 	GetUniqueName() string
 	GetContextAwareResources() []ContextAwareResource
 	GetBackgroundAudit() bool
+	GetSeverity() (string, bool)
+	GetCategory() (string, bool)
```
Comment (flavio) [inline pkg/apis/policies/v1/policy.go:107]: Does this enrich the output of `kubectl get [type]policy` by adding a dedicated column?

**Review by flavio — APPROVED** [review]

File: pkg/apis/policies/v1/policy.go (hunk 1 above)
Comment (viccuad) [reply pkg/apis/policies/v1/policy.go:109]: Done in https://github.com/kubewarden/kubewarden-controller/pull/462/commits/423cefd71723bf3606737586d0cfd2a79cb0c301

**Review by flavio — APPROVED** [review]

**Comment by viccuad** [comment]
> > I tried to document the new annotations in the crd generation docs, but alas, I found no way to get the metadata annotations listed.
> 
> Let's add something in our docs website then.

I have added that task to the doc task of the epic, to move this forward.

**Pull request merged**